src/contexts/states/app.ts

```diff
@@ -22,6 +22,7 @@ export const initApp: State.App = {
   appWidth: window.innerWidth,
   appHeight: window.innerHeight,
   language: navigator.language.includes('zh') ? 'zh' : 'en',
+  isMainnet: false,
```

Comment:
when to change this value to true?

Reply:
`isMainnet` will be changed when header add dropdown component to switch mainnet and testnet

Reply:
`isMainnet` default value is false, what happens if I visit the explorer through testnet url?

Reply:
Now we don't release mainnet version, so I just keep `isMainnet` false to make this feature has no effect to explorer production and this feature need hide now.

When we need release miannet and testnet,  change these parameters and config will be very quick.